### ISSUE TITLE
cannot access '<country>*': No such file or directory

### ISSUE THREAD

**Issue by aawobdev** [ilteoood/docker-surfshark#72]
I had this issue when updating my image to latest.

I think this line in the startup.sh line 9:
`VPN_FILE=$(ls "${SURFSHARK_COUNTRY}"* | grep "${SURFSHARK_CITY}" | grep "${CONNECTION_TYPE}" | shuf | head -n 1)`

This causes the error. This line should be (I think):
`VPN_FILE=$(ls -d *"${SURFSHARK_COUNTRY}"* | grep "${SURFSHARK_CITY}" | grep "${CONNECTION_TYPE}" | shuf | head -n 1)`

Note the additional * due to the vpn config files not starting with the country (not mine anyway) AND the -d option added.

e.g. rfr_uk-lon.prod.surfshark.com_tcp

**Comment by dd32** [comment]
Yep, as with #71 the file structure of the ZIPs Surfshark provides has changed.

most specific change that would work is this:
```diff
-VPN_FILE=$(ls "${SURFSHARK_COUNTRY}"* | grep "${SURFSHARK_CITY}" | grep "${CONNECTION_TYPE}" | shuf | head -n 1)
+VPN_FILE=$(ls rgb_"${SURFSHARK_COUNTRY}"* | grep "${SURFSHARK_CITY}" | grep "${CONNECTION_TYPE}" | shuf | head -n 1)
```

Using `*country*` would work, but would cause problems for some countries, such as `AT` where it would then also match `gr-ath` and `us-atl` instead of just `at-vie`.

**Comment by ilteoood** [comment]
This has been fixed with release 1.7.1
